src/Components/Login/Login.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import style from './Login.module.scss'
+import aboutIcon from '../../assets/navbar/aboutNavIcon.png'
+import settingsIcon from '../../assets/navbar/settingsNavIcon.png'
+import statsIcon from '../../assets/navbar/statsNavIcon.png'
+import timerIcon from '../../assets/navbar/timerNavIcon.png'
+
+const Login = () => {
+  return (
+    <section className={style.loginContainer}>
+      <h2 className={style.loginHeader}>Welcome to Som Timer</h2>
+      <p className={style.appDescription}>
+        Som Timer is a timer that cares. This application serves curated wellness content on break intervals. Be mindful of rest so you can focus best!
+        <br /><br />
+        Sign in with Google to get started.
+      </p>
+      {/* This login button comes from these docs https://developers.google.com/identity/sign-in/web/sign-in; We should explore these further for methods we'll need for Login */}
+      <div className='g-signin2' data-onsuccess='onSignIn' aria-label='Google Login Button'></div>
+      <section className={style.somTimerWalkthrough}>
+        <article className={style.walkthroughArticle}>
+          <img
+            className={style.icon}
+            src={timerIcon}
+            alt="timer icon"
+          />
+          <div className={style.walkthroughText}>
+            <h3 className={style.walkthroughHeader}>Home Page</h3>
+            <p className={style.walkthroughDescription}>On the home page, you can view your current focus or break interval in progress. Here, you'll find the countdown timer and built in media player that provides your wellness content.</p>
+          </div>
+        </article>
+        <article className={style.walkthroughArticle}>
+          <img
+            className={style.icon}
+            src={settingsIcon}
+            alt="settings icon"
+          />
+          <div className={style.walkthroughText}>
+            <h3 className={style.walkthroughHeader}>Settings Page</h3>
+            <p className={style.walkthroughDescription}>On the settings page, you can update your focus and break intervals to fit your needs. You can also adjust other preferences, like the sound of your notifications.</p>
+          </div>
```

Comment:
Do we want to add that you can also disable mood ratings? Not necessarily needed, but just a thought

Reply:
I think that would be nice to add in here!